pkg/storage/tsdb/block/markers.go

```diff
@@ -44,6 +44,7 @@ var (
 
 type Marker interface {
 	markerFilename() string
+	GetID() ulid.ULID
```

Comment:
in the go convention this should be named either `ID()` or `ULID()`.

And on top of this - since this is not the ID of the marker, but the ID of the block, should we call it something like `BlockULID()` or `BlockID()`?